com.transmissionbt.Transmission.appdata.xml

```diff
@@ -176,4 +177,7 @@
   </releases>
   <translation type="gettext">transmission-gtk</translation>
   <update_contact>will+transmission@willthompson.co.uk</update_contact>
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-info">moderate</content_attribute>
```

Comment:
Hmm, does this really apply?

Reply:
I gave the rationale in the commit message:

* moderate information sharing: Transmission sends up/download stats to
  the BitTorrent tracker, which I claim is "diagnostic data not
  identifiable to the user"

Reply:
Very happy to be disagreed with and remove this tag. It's not a clear fit…

Reply:
When I think of social info I think of things like real name, address, birthday etc.

I don't think IP address and download stats are social.

I'll leave the final call up to you just my thoughts.

Reply:
Yeah, I think the actual tag names in OARS can be quite misleading compared to the way they're expected to be used. Thus the questionaire I suppose. fwiw I would say that this should actually be mild, since it more closely fits the 'user counter' description of that state.

Reply:
Yep, let's split the difference and call it "mild". I think it would be wrong to say *no* information is shared.